conjure-object/Cargo.toml

```diff
@@ -11,8 +11,15 @@ readme = "../README.md"
 [dependencies]
 base64 = "0.13"
 chrono = { version = "0.4", features = ["serde"] }
+educe = { version = "0.4", default-features = false, features = [
+    "Hash",
+    "PartialEq",
+    "Eq",
+    "PartialOrd",
+    "Ord",
+] }
 lazy_static = "1.0"
-ordered-float = { version = "3.0", features = ["serde"] }
```

Comment:
Why did we drop the `.0` here?

Reply:
They're equivalent constraints - I think I just removed the dependency while writing the PR and then added it back later the other way.